### DISCUSSION TITLE
Neovim keybindings are not working with Kitty.

### DISCUSSION THREAD

**Discussion by gaurishhs** [kovidgoyal/kitty#6683] [Q&A]
Hi! I have been using kitty terminal emulator for a long time, But recently my keybindings related to (Cmd/Ctrl) have stopped working. (I did not change any configuration). The same thing works on Alacritty as it used to on Kitty. (Note: i have an empty kitty.conf)

https://github.com/kovidgoyal/kitty/assets/78847111/7a082c59-82da-4102-aca0-1ba8864153bc

**Comment by kovidgoyal** [comment]
That will be because neovim switched to using the kitty keyboar
dprotocol and it had various bugs in its implementation, although those
have all been fixed now as far as I know. Regardless this issue should
be reported to neovim.

**Reply by gaurishhs** [reply]
i see an open issue https://github.com/neovim/neovim/issues/14400 so there's nothing i can do atm to make that work?
